ports/openldap/portfile.cmake

```diff
@@ -9,20 +9,31 @@ vcpkg_extract_source_archive_ex(
     ARCHIVE "${ARCHIVE}"
     PATCHES
         openssl.patch
+        subdirs.patch
 )
 
-execute_process(
-    COMMAND autoreconf
-    WORKING_DIRECTORY ${SOURCE_PATH}
-)
+vcpkg_list(SET FEATURE_OPTIONS)
+if("tools" IN_LIST FEATURES)
+    vcpkg_list(APPEND FEATURE_OPTIONS --enable-tools)
+endif()
+
+# Disable build environment details in binaries
+set(ENV{SOURCE_DATE_EPOCH} "1659614616")
 
 vcpkg_configure_make(
     SOURCE_PATH "${SOURCE_PATH}"
+    AUTOCONFIG
     OPTIONS
+        ${FEATURE_OPTIONS}
         --disable-slapd
         --with-tls=openssl
         --without-cyrus-sasl
-        "LIBS=-ldl"

```

Comment:
I suspect this is still needed for CentOS-7 etc.

Reply:
I would like to know why it should be needed. 
(We know that it is needed for (static) openssl, and it is not in the list of libs which openldap would add on its own when running the check.)